rest-api.md

```diff
@@ -0,0 +1,767 @@
+## API
+
```

Comment:
Add details that all endpoints returns meta field with data (and data details)

Reply:
each endpoint contains this information

Reply:
But we can do it in one place and do not dublicate this info